include/fiction/algorithms/simulation/sidb/quickexact.hpp

```diff
@@ -199,6 +199,10 @@ class quickexact_impl
                     result.charge_distributions.push_back(charge_lyt_copy);
                 }
             }
+            else
```

Comment:
Is this important to you in any way? I am not a big fan of returning an empty charge distribution.

Reply:
I noticed `is_physically_valid` performs a check for the empty layout in each call which seems illogical as this function is called so often during simulation which performed on non-empty layouts in practice. Would you agree to that? I could also just make a case distinction for the empty layout in the simulators (and return no valid charge distributions) instead of this remedy